docs/source/glossary.md

```diff
@@ -17,4 +17,20 @@ Functional Programming
   This contrasts with functions or methods that depend on variables that are not explicitly passed as an input (such as accessing `self.variable` inside a method) or that alter the inputs or other state variables in-place, instead of returning new distinct variables as outputs.
```

Comment:
I think we can delete the sample terms already, right? @OriolAbril

Reply:
we can delete them once we start merging PRs, but I'd do that in a different PR that could also add the sorted flag, see https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=Glossary#directive-glossary